index.bs

```diff
@@ -394,9 +394,19 @@ spec:url; type:dfn; text:scheme
   <section algorithm="htmlportalelement-setsourceurl">
     To <dfn for="HTMLPortalElement">set the source URL of a <{portal}> element</dfn> |element|, run the following steps:
 
-    1. Let |guestBrowsingContext| be |element|'s [=guest browsing context=].
+    1. [=Assert=]: |element| is [=browsing-context connected=].
 
-    1. [=Assert=]: |guestBrowsingContext| is not null.
+    1. Let |hostBrowsingContext| be |element|'s [=node document|document=]'s [=browsing context=].
+
+    1. [=Assert=]: |hostBrowsingContext| is not null.
+
+    1. If |hostBrowsingContext| is a nested browsing context, then abort these steps.
+
+        <p class="note">
+          The user agent may choose to emit a warning if the author attempts to
+          use a <{portal}> element in a nested browsing context, as this is not
+          supported.
+        </p>
 
     1. If |element| has no <{portal/src}> attribute specified, or its value is the empty string,
         then [=close a portal element|close=] |element| and abort these steps.
```

Comment:
Drive-by nit: "return", not "abort these steps", for both.

Reply:
In general for all uses of the phrase "abort these steps", or is there some guidance for when each wording is preferred? If it's just an editorial change in general, maybe I should do a separate PR to globally replace the phrase.

Reply:
Yeah, probably best separate. The guidance is not very clear. We use "return" to return from an algorithm, but if that algorithm has sub-steps, e.g. in parallel or a queued task, then we have stuck with "abort these steps" (i.e. "return" is reserved for "exit the whole algorithm").

Reply:
In particular, https://infra.spec.whatwg.org/#algorithm-control-flow talks about using return, but it doesn't capture the remaining uses of "abort these steps" that we've evolved. I'll work on clarifying that in Infra.